src/m365/spp/commands/autofillcolumn/autofillcolumn-set.ts

```diff
@@ -0,0 +1,214 @@
+}
+
+interface CommandArgs {
+  options: Options;
+}
+
+const options = globalOptionsZod
+  .extend({
+    siteUrl: zod.alias('u', z.string()
+      .refine(url => validation.isValidSharePointUrl(url) === true, url => ({
+        message: `'${url}' is not a valid SharePoint Online site URL.`
+      }))),
+    listTitle: z.string().optional(),
+    listId: z.string()
+      .refine(value => validation.isValidGuid(value), listId => ({
+        message: `${listId} in parameter listId is not a valid GUID`
+      })).optional(),
+    listUrl: z.string().optional(),
+    columnId: zod.alias('i', z.string()
+      .refine(value => validation.isValidGuid(value), columnId => ({
+        message: `${columnId} in parameter columnId is not a valid GUID`
+      })).optional()),
+    columnTitle: zod.alias('t', z.string().optional()),
+    prompt: z.string().optional(),
+    isEnabled: z.boolean().optional()
+  })
+  .strict();
+
+declare type Options = z.infer<typeof options>;
+
+interface CommandArgs {
+  options: Options;
+}
+
+class SppAutofillColumnSetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.AUTOFILLCOLUMN_SET;
+  }
+
+  public get description(): string {
+    return 'Applies the autofill option to the selected column';
+  }
+
+  public get schema(): z.ZodTypeAny | undefined {
+    return options;
+  }
+
+  public getRefinedSchema(schema: typeof options): z.ZodEffects<any> | undefined {
+    return schema
+      .refine(options => [options.columnId, options.columnTitle].filter(x => x !== undefined).length === 1, {
+        message: `Specify exactly one of the following options: 'id' or 'title'.`
+      })
+      .refine(options => [options.listTitle, options.listId, options.listUrl].filter(x => x !== undefined).length === 1, {
+        message: `Specify exactly one of the following options: 'listTitle', 'listId' or 'listUrl'.`
+      });
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      if (this.verbose) {
+        await logger.log(`Applying an autofill column to a column...`);
+      }
+
+      const siteUrl = urlUtil.removeTrailingSlashes(args.options.siteUrl);
+      const listInstance = await this.getDocumentLibraryInfo(args);
+
+      if (listInstance.BaseType !== 1) {
+        throw Error(`The specified list is not a document library.`);
+      }
+
+      const column = await this.getColumn(args.options, listInstance.Id);
+
+      if (!Object.values(AllowedFieldTypeKind).includes(column.FieldTypeKind)) {
+        throw Error(`The specified column has incorrect type.`);
+      }
+
+      if (!!column.AutofillInfo) {
+        await this.updateAutoFillColumnSettings(args, column.Id, listInstance.Id, column.AutofillInfo);
+        return;
+      }
+
+      if (!args.options.prompt) {
+        throw Error(`The prompt parameter is required when setting the autofill column for the first time.`);
+      }
+
+      const requestOptions: CliRequestOptions = {
```

Comment:
For clarity lets add this request to a private function as well, just as the `updateAutoFillColumnSettings` function